.gitignore

```diff
@@ -63,4 +63,10 @@ dist
 !.elasticbeanstalk/*.global.yml
 
 # AWS DOcker build
-ecr.repo
\ No newline at end of file
+ecr.repo
+.DS_Store
```

Comment:
this all looks like global settings, rather than application specific settings, maybe due to vscode?
Worth therefore changing your `~/.gitconfig` and adding in ```[core]
	excludesfile = /<userhomedirectory>/.gitignore_global
	autocrlf = input
``` and then adding all the below to `~/.gitignore_global` instead

Reply:
Done, but left the old .project file